.gitignore

```diff
@@ -1,3 +1,5 @@
 # Binary of this program
 openebs-operator*
 /vendor/
+cover.out
```

Comment:
Next PR - Let us rename this repo to `oep-operator`

Reply:
https://github.com/mayadata-io/oep-operator/pull/5/
Renamed to `openebs-upgrade`.